specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json

```diff
@@ -1607,6 +1607,21 @@
         "kek": {
           "$ref": "#/definitions/JsonWebKey",
           "description": "The export key encryption Json web key. This key MUST be a RSA key that supports encryption."
+        },
+        "enc": {
+          "x-ms-client-name": "enc",
+          "type": "string",
+          "minLength": 1,
```

Comment:
If 1 value is required, should `enc` be a required property, as indicated in the `required` JSON schema array? If the property is required and 1 value is required, both should be authored.

Reply:
Same comment applies below.

Reply:
minLength is 1, I don't think that is the same as "required". The definition here matches other enum definitions in this file.